### DISCUSSION TITLE
Script failing when using image from shared image gallery

### DISCUSSION THREAD

**Discussion by athirasaseendran** [actions/virtual-environments#6662] [General questions]
Hi Team,
I have modified ubuntu2004.json file, such that it should fetch image from Shared Image gallery. Below is the modification made in that file . 
Added below lines in  builders section
"shared_image_gallery": {
        "subscription": "abc",
        "resource_group": "rgp",
        "gallery_name": "gallery",
        "image_name": "image",
        "image_version": "1.x"
},
			
"managed_image_name": "Ubuntu2004SharedImage",
"managed_image_resource_group_name": "rgp"

**Removed below lines from json file**
 "resource_group_name": "{{user `resource_group`}}",
 "storage_account": "{{user `storage_account`}}",
"capture_container_name": "images",
 "capture_name_prefix": "{{user `capture_name_prefix`}}", 
"image_publisher": "canonical",
"image_offer": "0001-com-ubuntu-server-focal",
"image_sku": "20_04-lts"

But its failing to start nginx. Below is the error 

azure-arm: Running tests.
    azure-arm:
    azure-arm: Running tests from '/imagegeneration/tests/WebServers.Tests.ps1'
    azure-arm: Describing Nginx
    azure-arm:   [+] Nginx CLI 166ms (138ms|28ms)
    azure-arm:   [-] Nginx Service 70ms (68ms|2ms)
    azure-arm:    Command 'sudo systemctl start nginx' has finished with exit code
    azure-arm:        Job for nginx.service failed because the control process exited with error code. See "systemctl status nginx.service" and "journalctl -
xe" for details.
    azure-arm:    at "sudo systemctl start nginx" | Should -ReturnZeroExitCode, /imagegeneration/tests/WebServers.Tests.ps1:19
    azure-arm:    at <ScriptBlock>, /imagegeneration/tests/WebServers.Tests.ps1:19
    azure-arm: Tests completed in 705ms
    azure-arm: Tests Passed: 1, Failed: 1, Skipped: 0 NotRun: 2

**Comment by mikhailkoliada** [comment]
Hello! This is the error connected with the template modification you've made, `SoftwareReport.Generator.ps1` is the script we use to generate the readme file

**Reply by athirasaseendran** [reply]
@mikhailkoliada I have updated the error. Its throwing exception for SoftwareReport.Generator.ps1
nginx is not starting after making template modification
